src/containers/VisitPage.js

```diff
@@ -125,20 +125,20 @@ class VisitPage extends Component {
     this.setState(prevState => ({
       currentStep: prevState.currentStep + 1
     }));
-  };
+  }
 
   previousStep = () => {
     this.setState(prevState => ({
-      currentStep: prevState.currentStep - 1
+      currentStep: prevState.currentStep 
```

Comment:
This breaks the 'Previous' buttons. I think we need to look into the tab-index of the elements instead.